iOS-Weather-App/Controllers/CurrentDetailedWeatherViewController/CurrentDetailedWeatherViewController.swift

```diff
@@ -0,0 +1,40 @@
+//
+// Created by pavels.garklavs on 23/02/2022.
+//
+
+import UIKit
+
+class CurrentDetailedWeatherViewController: UIViewController {
+
+    private let detailsTableView = UITableView()
+
+    override func viewDidLoad() {
+        setupTableView()
```

Comment:
when you override base function, don't forgot to override previous logic, should add super.viewDidLoad()